From d0c61e7e9b13185f424dff1f4ac697ec53089d69 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Tue, 4 Sep 2012 16:45:14 +0200
Subject: [PATCH] patch: sane-config-multilib

Squashed commit of the following:

commit 81aa4f41bf102b08258c8e1de1c0476835329ec5
Author: Nils Philippsen <nils@redhat.com>
Date:   Tue Sep 4 16:43:34 2012 +0200

    make installed sane-config multi-lib aware again

    This partially reverts commit 77c4ea1a7aa680fb1c3ee4daa1404f21439b2c9b.
---
 tools/sane-config.in | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tools/sane-config.in b/tools/sane-config.in
index 8e4b52a..1fae2e5 100644
--- a/tools/sane-config.in
+++ b/tools/sane-config.in
@@ -10,10 +10,6 @@ scriptname="sane-config"
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 
-# using our installed *.pc only - neither default nor user paths
-export PKG_CONFIG_LIBDIR="@libdir@/pkgconfig"
-export PKG_CONFIG_PATH=""
-
 pkgconfig_package=sane-backends
 
 usage ()
-- 
1.7.11.4

